src/PHPCR/Util/QOM/Sql2Scanner.php

```diff
@@ -159,7 +159,7 @@ protected function scan($sql2)
             $token = strtok(" \n\t");
         }
 
-        $regexp = '';
+        $regexp = array();
```

Comment:
@dbu i have no idea how this could work in the past :)

Reply:
i think old php versions are more tolerant about this. but i agree it was wrong.